generalFunctionsAndVariables.js

```diff
@@ -0,0 +1,276 @@
+/*
+	Functions and variables segmented into:
+		Global Variables
+		Dropdown Menu functions and variables
+		Custom User Input Variables
+		Data Parsing
+		Time Operations
+*/
+
+/*
+	Global Variables
+		graphNumberLimit
+		graphDisplayUpperLimit
+		propertiesList
+		dropdownVariables
+		var lastDataPull;
+		// Global variables for referencing
+			deviceList
+			totalNumUsers
+			eventSelect
+			eventGraph
+		initialSettings()
+*/
+// Limiting variables for graph display
+var graphNumberLimit = 20; // maximum number of items for pie chart (switches 
+							//to bar chart after)
+var graphDisplayUpperLimit = 100; // maximum number for display regardless of 
+									//custom input
+
+// Set initial properties required for segmentation
+var propertiesList = {
+	appNameString : null,
+	platformString : null,
+	deviceModelString: null,
+	from: moment().subtract(1, 'months').format("YYYY-MM-DD"),
+	to: moment().format("YYYY-MM-DD"),
+	limit : 10
+}
+var dropdownVariables = {
+	platform : { title : 'Platform', ID : '.chosen-platform', 
+		stringName : "platformString"},
+	appName : { title : 'App Name', ID : '.chosen-appName', 
+		stringName : "appNameString"},
+	model : { title : '$model', ID : '.chosen-model', 
+		stringName : "deviceModelString"}
+};
+
+// Store data from last pull (for use with limit changes)
+var lastDataPull;
+// Global variables for referencing
+var deviceList;
+var totalNumUsers;
+var eventSelect;
+var eventGraph = $('#graph');
+
+function initialSettings(){

```

Comment:
Is this initialSettings or initializeSettings?  I would expect initialSettings to be just a read-only data structure that lists all of your settings.  I would expect initializeSettings to actually change the settings to match the initial.

Or is this really initializeChosenInterface?